packages/klevu-ui/storybookAddons/klevuSettings.tsx

```diff
@@ -39,18 +39,21 @@ export const Tool = memo(function KlevuSettings() {
   const [apiKey, setApiKey] = useState(localStorage.getItem("klevu-api-key") || "")
   const [url, setUrl] = useState(localStorage.getItem("klevu-url") || "")
   const [language, setLanguage] = useState(localStorage.getItem("klevu-language") || undefined)
+  const [useKMC, setUseKMC] = useState<boolean>(localStorage.getItem("klevu-use-kmc") === "false" ? false : true)
 
   const onSave = (e) => {
     localStorage.setItem("klevu-api-key", apiKey)
     localStorage.setItem("klevu-url", url)
     localStorage.setItem("klevu-language", language ?? "")
+    localStorage.setItem("klevu-use-kmc", useKMC.toString() ?? "true")
     window.location.reload()
   }
 
   const onClose = () => {
     setApiKey(localStorage.getItem("klevu-api-key") || "")
     setUrl(localStorage.getItem("klevu-url") || "")
     setLanguage(localStorage.getItem("klevu-language") || undefined)
+    setUseKMC(localStorage.getItem("klevu-use-kmc") === "false" ? false : true)
```

Comment:
is this value being used when we do the init? I dont see references to this key in this PR. think you have used the wrong key in `storybook-utils.ts`

Reply:
Good find!